nimble/controller/include/controller/ble_ll_scan.h

```diff
@@ -97,9 +97,9 @@ struct ble_ll_scan_params
 #define BLE_LL_AUX_TRUNCATED_SENT       0x40
 #define BLE_LL_AUX_HAS_ADI              0x80
 
-#define BLE_LL_SET_AUX_FLAG(aux_data, flag) ((aux_data)->flags |= flag)
-#define BLE_LL_CLEAR_AUX_FLAG(aux_data, flag) ((aux_data)->flags &= ~flag)
-#define BLE_LL_CHECK_AUX_FLAG(aux_data, flag) (!!((aux_data)->flags & flag))
+#define BLE_LL_AUX_SET_FLAG(aux_data, flag) ((aux_data)->flags |= flag)
+#define BLE_LL_AUX_CLEAR_FLAG(aux_data, flag) ((aux_data)->flags &= ~flag)
+#define BLE_LL_AUX_CHECK_FLAG(aux_data, flag) (!!((aux_data)->flags & flag))
```

Comment:
since you modify those macros anyway, it would be safer to use `(flag)` instead of `flag` and there's no need for `!!` as it seems to be used only in `if` statements (?)

Reply:
agree